.travis.yml

```diff
@@ -123,30 +124,30 @@ script:
   fi
 
 - if [[ $TRAVIS_TAG ]]; then
-    conda build -q tools/conda.recipe;
-    (cd python-qcgrids; conda build -q tools/conda.recipe);
+    BUILD_TYPE="release" conda build -q tools/conda.recipe;
+    (cd python-qcgrids; BUILD_TYPE="release" conda build -q tools/conda.recipe);
     conda install --use-local qcgrids python-qcgrids;
     ${CONDA_PREFIX}/libexec/qcgrids/test_qcgrids;
+
     (cd; nosetests qcgrids -v --detailed-errors);
-    conda uninstall python-qcgrids;
-    conda uninstall qcgrids;
   else
-    conda install -c theochem cellcutoff python-cellcutoff;
-    (mkdir debug; cd debug; cmake cmake -DCMAKE_BUILD_TYPE=debug ..; make);
-    (cd debug; DYLD_LIBRARY_PATH=${DYLD_LIBRARY_PATH}:${CONDA_PREFIX}/lib/ ./qcgrids/tests/test_qcgrids);
-
-    if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
-      cardboardlinter --refspec $TRAVIS_BRANCH -f 'dynamic';
-    fi
+    BUILD_TYPE="debug" conda build -q tools/conda.recipe;
+    (cd python-qcgrids; BUILD_TYPE="debug" conda build -q tools/conda.recipe);
+    conda install --use-local qcgrids python-qcgrids;
+    ${CONDA_PREFIX}/libexec/qcgrids/test_qcgrids;
 
-    (cd python-qcgrids; python setup.py build_ext -i --define CYTHON_TRACE_NOGIL);
     (cd python-qcgrids; LD_LIBRARY_PATH=$LD_LIBRARY_PATH:../debug/qcgrids nosetests qcgrids -v --detailed-errors --with-coverage --cover-package=qcgrids --cover-tests --cover-erase --cover-inclusive --cover-branches; coverage xml -i);
 
     if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
+      cardboardlinter --refspec $TRAVIS_BRANCH -f 'dynamic';
       (cd python-qcgrids; LD_LIBRARY_PATH=$LD_LIBRARY_PATH:../debug/qcgrids cardboardlinter --refspec $TRAVIS_BRANCH -f 'dynamic');
     fi
   fi
 
+# Keep conda cache clean
+- conda uninstall python-qcgrids;
+- conda uninstall qcgrids;
+
```

Comment:
You can uninstall both in one call, which will save a few seconds. Not a big deal.